README.md

```diff
@@ -114,21 +69,54 @@ where `bin/`, `lib/` etc. are. This should be the only argument that is
 mandatory, everything else should be optional. The default build type is
 Release, though this can be overridden.
 
-The following example shows how to specify the ComputeCpp location, then
-build and run the tests.
+The following command shows how to compile SYCL-DNN.
 
 ```bash
 # Setup build environment
 mkdir build && cd build
 cmake .. -DComputeCpp_DIR=/path/to/computecpp
 # Compile SYCL-DNN
 make -j`nproc`
+```
+
+### Sample Code
+
+The "samples" directory contains sample code for the 2D convolution and pooling 
+operations offered by SYCL-DNN. These binaries are compiled when building SYCL-DNN 
+using CMake.
+
+### Running the SYCL-DNN Tests
+
+The SYCL-DNN tests are compiled when building SYCL-DNN using CMake.
+The following command shows how to run the tests.
+
+```bash
```

Comment:
I don't really understand the addition of these two sections. I don't really think they're necessary. The parts of the commands listed below are all labelled, which I think is a little clearer.

Reply:
My rationale is that the first thing people tend to want to do is look at any sample code that is available for a library, I think making it clear where it is and what it is demonstrating is a good thing. Likewise with the test code, someone might want to check what works on their device using the test code and this makes it totally clear how to run these.